SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,64 @@
+---
+to: Student Robotics 2023
+subject: SR2024 Competition Details
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
+
+Thanks to everyone who has already provided their attendance details. If you haven't done that yet, or things have changed substantially please [provide attendance details](https://forms.gle/8YhfdRbfDWP3KKNy8).
+
+Doors open on both days at 9am and each day will finish around 5:30pm. The event will formally start at 10:30am on Saturday. There are now full day schedules on the [event page][competition-event-page]. **If you are expecting to arrive after 10am on Saturday, please let us know.**
+
+You'll have access to your pits and the test arena when you arrive, giving your team time to work on their robot before things get going in earnest.
+
+You're welcome to bring family and friends as guests, however please note that space in the pits is limited.
+
+We'll be publishing rules for behaviour in/around the competition venue when you arrive. If you have any questions or concerns beforehand, please let us know.
+
+The competition this year is a public event, and we do not have exclusive access to the venue.
+
+If you have any issues on the day, we'll be monitoring the `teams@` email address and our Discord server.
+
+## Photos & Videos
+
+Throughout the competition, Student Robotics volunteers will be taking photos and videos for the purpose of prompting and marketing the charity. The arena will also be [livestreamed](https://studentrobotics.org/events/sr2024/competition/#livestream) on our YouTube channel. Unlike previous competitions, we will not be issuing Media Consent forms.
+
+If you or your competitors would not like to appear in our photos and videos, please let our reception desk know when you arrive.
+
+## Equipment
+
+Please ensure that your team bring at least one laptop or appropriate device with them to the competition to access the docs and work on your robot's code. We are unable to provide any computing devices for competitor use at the competition. We will be providing a wireless internet connection to all teams.
+
+Any tools that your team requires to modify their robot must be brought to the competition by the team: we cannot provide any. In particular, the use of any dangerous tools must be supervised by the responsible adult for the team. Power tools may only be used in the designated "Power Tools" area.
+
+You must bring all of the robotics kit (including the white box it came in) to the competition, so that we can collect it at the end of the event. We'll be taking both your batteries, the battery charger and battery bag off you when you arrive, so please make sure those are easily accessible. A list of the kit to return is attached to this email.
```

Comment:
TODO: create & render that attachment, plus link to it in the front-matter.

See https://studentrobotics.slack.com/archives/CMQ49PXPG/p1710518042073149?thread_ts=1710517438.705659&cid=CMQ49PXPG.

Reply:
https://github.com/srobo/team-emails/pull/183/commits/43060353cd2930610afc2a0269b4fdd8dc619f77